service/history/api/describeworkflow/api.go

```diff
@@ -47,8 +47,8 @@ import (
 	"go.temporal.io/server/service/history/api"
 	"go.temporal.io/server/service/history/shard"
 	"go.temporal.io/server/service/history/workflow"
+
```

Comment:
nit: remove all added empty lines in imports

Reply:
Moved inside RetryActivity

Reply:
actually will move outside. RetryActivity is not the right place for it. I refactor the code to properly update mutableState